### PR TITLE
Remove i18n setup

### PR DESCRIPTION
**Pull request by rmnprkrl**
This PR removes i18n setup to support docs structure revamp and its maintenance. This step is part of set of upcoming tasks aiming to further clean up Gatsby infrastructure to improve docs integration. i18n will be reimplemented back globally as part of substrate.io platform.

Notes:
- [x] removed all gatsby and react i18n modules and config
- [x] removed `LocalizedLink` from templates (should be part of Link component in next implementation)
- [x] all nav strings directly in `src/components/DevNavMenu.tsx`
- [x] fixing How-to Guides capitalization

### CODE REVIEW

File: v3/docs/00-style-and-contributor-guidelines/b-main-style-guide/index.mdx [hunk 1]
```diff
@@ -429,10 +429,8 @@ your life and the maintainers lives easier.
 
 ### New pages and moving files
 
-- The `i18n` folder is used for translation **but also for the rendering of the navigation**
-  **elements sections**. Your `index.mdx` file's `title` is not the source for this.
 - If you are adding or renaming a page, you **must** add it correctly in `src/components/DevNavMenu.tsx` and
```
Comment (rmnprkrl) [inline v3/docs/00-style-and-contributor-guidelines/b-main-style-guide/index.mdx:432]: @sacha-l there will be one step less needed now when adding new material as it does not source translation strings anymore. All is now in `src/components/DevNavMenu.tsx`

I've updated this guide to reflect the change.

**Review by imadarai — APPROVED** [review]
Thanks @rmnprkrl ! 
Looks good to me.  I cloned the repo locally to go over each page and make sure everything is working as intended.  

@sacha-l I checked through the `DevNavMenu.tsx` and make sure all the strings are copied over correctly.  Would be nice to have another pair of eyes for a general spell check.

**Review by sacha-l — COMMENTED** [review]
Did a pass on checking the side nav for spelling. LGTM.

One thing that stood out is the breadcrumb nav when on this page: /how-to-guides/v3/
... it should read "How-to Guides" in the breadcrumb.  I know it's not part of this PRs purpose, but maybe we can wedge that in. I'll leave it up to you, happy to make a separate issue or PR if that help. Lmk.

**Comment by rmnprkrl** [comment]
@sacha-l fixed the lowercase "How-to"

**Pull request merged**